cloud: make community access visible and accurate in connected care relevant… (#22346)
pingcap/docs: tidb-cloud/tidb-cloud-support.md

---
title: Working with Support Team
summary: Learn how to contact the support team of TiDB Cloud.
---

> # Working with Support Team

Reviewer: ```suggestion
# Work with the Support Team
```

> TiDB Cloud offers tiered support plan offerings tailored to meet customers' needs. This document describes how to send support requests through the PingCAP Help Center portal. If you are looking for more information about our support offerings, see [Connected Care Details](/tidb-cloud/connected-care-detail.md).

Reviewer 2: ```suggestion
TiDB Cloud offers tiered support plans to meet your needs. This document describes how to submit support requests through the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals). For more information about support plans, see [Connected Care Details](/tidb-cloud/connected-care-detail.md).
```
